src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event extends Task {
+    protected String start;
+    protected String end;
+
+    public Event(String description,  String start, String end) {
```

Comment:
```suggestion
    public Event(String description, String start, String end) {
```
I think there was a minor typo where you used 2 spaces to separate non-whitespace characters instead of 1